singleWindow.html

```diff
@@ -0,0 +1,327 @@
+<!doctype html>
+<html>
+    <head>
+        <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+        <meta content="utf-8" http-equiv="encoding">
+        <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+        <link rel="stylesheet" type="text/css" href="css/annotation.css">
+        <link rel="stylesheet" type="text/css" href="css/font-awesome.min.css">
+        <link rel="stylesheet" type="text/css" href="css/jqcloud.css">
+        <script src="scripts/jquery.js"></script>
+        <script src="scripts/jquery.awesomeCloud-0.2.js"></script>
+        <title>
+            All Context Screen
+        </title>
+    </head>
+    <body>
+        <table width="100%" style="max-height: 200px; ">
+            <tr class="spaceUnder">
+                <td style="text-align: center;">
+                    <div id ="horizontal_line" style="margin-bottom: 5px"></div>
+                        <div class='container' style="background: white">
+                            <img class='resize_fit_center'
+                            src='images/logo.gif' />
+                        </div>
+                    <div id="horizontal_line" style="margin-top:3px"></div>
+                </td>
+            </tr>
+        </table>
+        <div>
+            <h2 style="text-align: center;"><strong>Alexa, Whois overview of Current URL</strong><br></h2>
+        </div>
+        <div class="container" id="container-whole">
+            <div class="row">
+                <div class="col-sm-6">
+                    <h4></h4> 
+                    <pre id="show_alexa_data" style="font-size: 20px">  
+                    </pre>
+                </div>
+                <div class="col-sm-6">
+                    <h4></h4> 
+                    <pre id="show_whois_data" style="font-size: 20px">
+                    </pre>
+                </div>
+            </div>
+            <div></div>
+        </div>
+        <div>
+                <h2 style="text-align: center;"><strong>Wayback Machine overview of Current URL</strong></h2>
+        </div>
+        <BR>
+        <BR>
+        <div class="container">
+            <div class="row">
+                <div class="col-sm-6">
+                    <strong style="font-family: Bebas Neue;font-size: 30px;">Date of First Archive</strong>
```

Comment:
@anishsarangi since you use this style in multiple elements, why don't use create a new css class and just apply it in all elements? This comment can be also applied everywhere in this PR. I think that its not good at all to have SO many `style=""` attributes everywhere setting again and again the same CSS attributes. This approach cannot scale.

Reply:
Thank you @vbanos . I will format the code correctly in the next commit.